linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -138,6 +142,82 @@ where
     }
 }
 
+/// A data structure to store the current state of available ports.
```

Comment:
At a high level, this is generally not the proper way to obtain a free port in a system.